src/base/system_parameter.c

```diff
@@ -6017,13 +6017,13 @@ static SYSPRM_PARAM prm_Def[] = {
    (char *) NULL,
    (DUP_PRM_FUNC) NULL,
    (DUP_PRM_FUNC) NULL},
-  {PRM_ID_JAVA_STORED_PROCEDURE_RESERVE_01,
-   PRM_NAME_JAVA_STORED_PROCEDURE_RESERVE_01,
-   (PRM_FOR_SERVER | PRM_HIDDEN),
+  {PRM_ID_JAVA_STORED_PROCEDURE_UDS,
+   PRM_NAME_JAVA_STORED_PROCEDURE_UDS,
+   (PRM_FOR_SERVER),
```

Comment:
Since Windows can use only TCP and UNIX (LINUX) uses UDS by default, it would be better to set it as a hidden property and specify TCP only for specific test purposes.